pkg/storage/mvcc_test.go

```diff
@@ -6255,6 +6256,344 @@ func TestMVCCExportToSSTSErrorsOnLargeKV(t *testing.T) {
 	require.ErrorAs(t, err, &expectedErr)
 }
 
+// TestMVCCExportFingerprint verifies that MVCCExportFingerprint correctly
+// fingerprints point keys in a given key and time interval, and returns the
+// range keys in a pebble SST.
+//
+// This test uses a `fingerprintOracle` to verify that the fingerprint generated
+// by `MVCCExportFingerprint` is what we would get if we iterated over an SST
+// with all keys and computed our own fingerprint.
+func TestMVCCExportFingerprint(t *testing.T) {
```

Comment:
Can you add some test cases to `TestMVCCHistories` as well? We already have comprehensive export tests there in `mvcc_histories/export`, including an example dataset (but let's use a separate file for fingerprint tests).

Reply:
Nice! Didn't know this test existed.

I added the same tests as in `mvcc_histories/export` but instead of printing point keys I validate the fingerprint against an oracle. Apologies if this file is hard to review but I ran a diff with some sed between `mvcc_histories/export` and `mvcc_histories/export_fingerprint` and the rangekeys outputted in both are identical as expected.

Reply:
Thanks! These should output the actual fingerprint as well, to assert the stability of the fingerprint. I don't think it's necessary to add `pointkeys_fingerprint_verified`, since the test will fail if it doesn't match, but no strong opinion.

I have a slight preference for adding a `fingerprint` option to the existing `export` command rather than adding a new command. Essentially, `if e.hasArg("fingerprint") { ... }`. In any case, the command list comment at the top of the file needs to be updated.

Reply:
good point, changed to what you recommended.